### PR TITLE
Parse -Xlog:gc variants for compatibility

### PR DESCRIPTION
**Pull request by pshipton**
Issue https://github.com/eclipse/openj9-docs/issues/671

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

File: docs/cmdline_migration.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ You can use the following command-line options in OpenJ9, just as you did in Hot
 | [`-Xcheck:jni`](xcheck.md)                                       | Runs additional checks for JNI functions during VM startup.                                                                                  |
 | [`-Xfuture`](xfuture.md)                                         | Turns on strict class-file format checks.                                                                                                    |
 | [`-Xint`](xint.md)                                               | Runs an application in interpreted-only mode.                                                                                                |
+| [`Xlog`](xlog.md)                                                | Some forms of `-Xlog` which enable GC logging are recognized. (Equivalent to [`-Xverbosegclog`](xverbosegclog.md)).
```
Comment (SueChaplain) [inline docs/cmdline_migration.md:42]: Link name missing "-" ........ `-Xlog`

Comment (SueChaplain) [reply docs/cmdline_migration.md:42]: -> Some forms of `-Xlog` *that* enable GC logging are recognized.

Comment (pshipton) [reply docs/cmdline_migration.md]: Updated to fix both.

File: docs/xlog.md [hunk 2]
```diff
@@ -24,7 +24,17 @@
 
 # -Xlog 
 
-To control logging to the operating system message log, use [`-Xsyslog`](xsyslog.md). When [`-XX:-LegacyXlogOption`](xxlegacyxlogoption.md) is set, which is the default, the `-Xlog` option is not recognized. When `-XX:+LegacyXlogOption` is set, `-Xlog` is equivalent to `-Xsyslog`.
+To control logging to the operating system message log, use [`-Xsyslog`](xsyslog.md). When [`-XX:-LegacyXlogOption`](xxlegacyxlogoption.md) is set, which is the default, the `-Xlog` option is processed for compatibility as described below. When `-XX:+LegacyXlogOption` is set, `-Xlog` is equivalent to `-Xsyslog`.
```
[the remark this answers is not in the document]
Comment (pshipton) [reply docs/xlog.md:27]: `-XX:+LegacyXlogOption` is only the default for IBM Java 8, not for OpenJ9.

[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/xlog.md:27]: OK - will address that elsewhere. Perhaps we can just remove between the commas. Updated next suggestion too.

[the remark this answers is not in the document]
Comment (pshipton) [reply docs/xlog.md]: I've pushed updates to address this comment.

File: docs/version0.24.md [hunk 3]
```diff
@@ -45,9 +45,9 @@ OpenJ9 release 0.24.0 supports OpenJDK 8, 11, and 15. Binaries are available fro
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
-### `-Xsyslog` replaces `-Xlog` option by default
+### `-Xsyslog` replaces `-Xlog` option by default, `-Xlog` behavior change
```
[the remark this answers is not in the document]
Comment (pshipton) [reply docs/version0.24.md:48]: I've pushed updates to address this comment. Not sure how to update the TOC. If you want to push and take over that is fine with me. Also happy to add additional changes.

[the remark this answers is not in the document]
Comment (pshipton) [reply docs/version0.24.md:48]: Found the TOC, will push that change shortly.

[the remark this answers is not in the document]
Comment (pshipton) [reply docs/version0.24.md:48]: Pushed the TOC change.

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Pull request merged**